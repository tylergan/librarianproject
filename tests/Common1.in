ADD MEMBER Amelia Earhart
ADD MEMBER Tom Clancy
COMMON 100000 100001 
ADD COLLECTION books.csv
COMMON 100000 100001 
RENT 100000 111111
RENT 100000 111112
RENT 100000 111113
RENT 100001 111111
RELINQUISH ALL 100000
RENT 100001 111112
RENT 100001 111113
RELINQUISH ALL 100001
COMMON 100000 100001
EXIT