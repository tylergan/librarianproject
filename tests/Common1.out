user: Success.

user: Success.

user: No books in system.

user: 17 books successfully added.

user: No common books.

user: Success.

user: Success.

user: Success.

user: Book is currently unavailable.

user: Success.

user: Success.

user: Success.

user: Success.

user: The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)

user: Ending Library process.
